lib/tbot/config/service_workload_identity_aws_ra.go

```diff
@@ -0,0 +1,170 @@
+// Teleport
+// Copyright (C) 2025 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package config
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws/arn"
+	"github.com/gravitational/trace"
+	"gopkg.in/yaml.v3"
+
+	"github.com/gravitational/teleport/lib/tbot/bot"
+)
+
+const (
+	WorkloadIdentityAWSRAType        = "workload-identity-aws-ra"
```

Comment:
super nit picky: Is `ra` a common way of abbreviating "roles anywhere"?

Given this type name is still quite long (24 chars), I wonder if it's really worth the ambiguity/loss of google-ability. Especially given we're spending 17 of those characters on `workload-identity` 😄

Reply:
I wouldn't say it's super common - I think this is a fair enough comment, I'll switch to using `roles-anywhere` in its full form.

Reply:
[68047c3](https://github.com/gravitational/teleport/pull/52426/commits/68047c363d08eb7d86bbe860c474bb8d3c326111)